tests/test_format.py

```diff
@@ -102,7 +102,7 @@ def test_strip_comments_preserves_linebreak(self):
         assert res == 'select *\n\nfrom foo'
 
     def test_strip_ws(self):
-        f = lambda sql: sqlparse.format(sql, strip_whitespace=True)
+        def f(sql): return sqlparse.format(sql, strip_whitespace=True)
```

Comment:
ah, I've got a different opinion on this. E731 states to prefer a function definition over assigning a lambda expression to a variable. But as also stated in E731 the primary reason is for debugging as the lambda expression shows up as `<lambda>` in a traceback. 

But here the lambda is just a shorthand for the formatting code to make the tests shorter and more readable. Iff debugging is needed here, there's only one `<lambda>`. So it's pretty obvious. 

On the other hand I find single line function definitions a bit harder to read and I would avoid them if possible.

So to me "Readability counts" is the argument here, especially as the one reason for using function definitions instead of assigning a lambda function to a variable doesn't count here.

Reply:
Okay, I get you!)
I will close this PR now,
and thank you for your feedback